moleculekit/tools/crystalpacking.py

```diff
@@ -83,17 +115,19 @@ def viewCrystalPacking(mol, hexagonal=False, style_display='NewCartoon', viewerh
 
     caux = (np.cos(alpha) - np.cos(beta) * np.cos(gamma)) / np.sin(gamma)
     axes = np.array([[a, 0, 0], [b * np.cos(gamma), b * np.sin(gamma), 0], [c * np.cos(beta), c * caux,
-                           c * np.sqrt(1 - np.cos(beta) ** 2 - caux ** 2)]])
+                    c * np.sqrt(1 - np.cos(beta) ** 2 - caux ** 2)]])
     size = np.array([axes[0][0], axes[1][1], axes[2][2]])
 
     molunit = Molecule()
 
-    _draw_cell(axes, ci['sGroup'], viewerhandle, hexagonal=hexagonal)
+    if visualize:
+        _draw_cell(axes, ci['sGroup'], viewerhandle, hexagonal=hexagonal)
```

Comment:
Can you move this to the end of the function so that it visualizes the cell on top of the last Molecule?

Reply:
Done.